app/controllers/vm_common.rb

```diff
@@ -573,15 +585,9 @@ def snap_vm
       @name = params[:name]
       @description = params[:description]
       if params[:name].blank? && !@record.try(:snapshot_name_optional?)
-        add_flash(_("Name is required"), :error)
```

Comment:
tnx. I just discovered I have a Vim plugin to extract methods ))
Although it did put the method inside a non related if block at first, but it was still nice :)